driver/src/main/com/mongodb/MongoDatabaseImpl.java

```diff
@@ -296,12 +297,12 @@ public void createView(final ClientSession clientSession, final String viewName,
     @Override
     public void createView(final ClientSession clientSession, final String viewName, final String viewOn,
                            final List<? extends Bson> pipeline, final CreateViewOptions createViewOptions) {
+        notNull("clientSession", clientSession);
```

Comment:
This would have been caught by the existing unit test, had I actually run that test before pushing.  Sorry about that.